### ISSUE TITLE
vg construct from fasta path name selection could be better.

### ISSUE THREAD

**Issue by JervenBolleman** [vgteam/vg#547]
When building a VG from an Ensembl Bacteria the selected path name is not ideal.

The FASTA header is
```
>Chromosome dna:chromosome chromosome:ASM1942v1:Chromosome:1:4686137:1 REF
```
This gives after VG construct

```
vg construct --reference ecoli/Escherichia_coli_str_k_12_substr_dh10b.ASM1942v1.dna.chromosome.Chromosome.fa -m 1000 > ecoli/Escherichia_coli_str_k_12_substr_dh10b.ASM1942v1.vg
```

The following path name
```
 vg view ecoli/Escherichia_coli_str_k_12_substr_dh10b.ASM1942v1.vg | grep "^P" | head -n 1

P       1       Chromosome      1       +       1000M
```

This is not ideal, as then all paths if building a VG of Ensembl bacteria have the same name.

Desired resolution, entire fasta headers should be preserved by vg construct

**Comment by JervenBolleman** [comment]
This is tricky because paths.cpp/hpp use the path name as identifier for the path object. Collisions here will break things.

**Comment by JervenBolleman** [comment]
I had this error that might be due to this.
```
graph path 'Chromosome' invalid: edge from 149958 end to 326110 start does not exist
creating edge
```
I renamed the paths by hand and am rerunning the commands and see if that makes a difference.

**Comment by adamnovak** [comment]
I think we probably want to keep the behavior we have here; it's in line with the FASTA spec (to the point that there is a spec). The first whitespace-separated token in the FASTA header is the unique sequence ID, while everything after that is the "description".

We could work out a way to keep the description with the path, but we probably want it to be identified by FASTA ID.

Maybe we could add support for quoted FASTA IDs? I don't know if anyone else does that.

Why is Ensembl Bacteria producing such weird FASTAs? It really looks like the third token, `chromosome:ASM1942v1:Chromosome:1:4686137:1`, ought to be the ID.

**Comment by JervenBolleman** [comment]
Makes sense, we then just need code to warn during alignment that a path name has already been taken.
